src/Console/Commands/CrudBackpackCommand.php

```diff
@@ -52,14 +52,14 @@ public function handle()
         }
 
         // Create the CRUD Model and show output
-        $this->call('backpack:crud-model', ['name' => $name]);
+        $this->call('backpack:crud-model', ['name' => $nameTitle, '--force' => $this->option('force')]);
```

Comment:
We can probably store the `--force` option up close where we init other command properties like `$name`, `$nameTitle` etc. We then re-use the variable across the command 👍 This is just personal preference, it's not that is bad implemented.

```suggestion
        $this->call('backpack:crud-model', ['name' => $nameTitle, '--force' => $force]);
```

Reply:
Please also take a look at this @karandatwani92 let me know if you think we shouldn't do this, otherwise please do it 👍